extensions/cricketcast/src/index.jsx

```diff
@@ -0,0 +1,74 @@
+import { ActionPanel, OpenInBrowserAction, List, showToast, ToastStyle, CopyToClipboardAction } from "@raycast/api";
+import { useEffect, useState } from "react";
+import convert from "xml-js";
+import fetch from "node-fetch";
+
+function Actions({ item }) {
+  return (
+    <ActionPanel title={item.title}>
+      <ActionPanel.Section>{item.link && <OpenInBrowserAction url={item.link} />}</ActionPanel.Section>
+      <ActionPanel.Section>
+        {item.link && (
+          <CopyToClipboardAction content={item.link} title="Copy Link" shortcut={{ modifiers: ["cmd"], key: "." }} />
+        )}
+      </ActionPanel.Section>
+    </ActionPanel>
```

Comment:
Better only having one ActionPanel.Section, then will the second item automatically get a modifier